pom.xml

```diff
@@ -104,6 +104,20 @@
         <artifactId>maven-enforcer-plugin</artifactId>
         <version>3.2.1</version>
         <executions>
+          <execution>
+            <id>enforce-maven-version</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireMavenVersion>
+                  <!-- Usage of `.mvn/jvm.config` for Error Prone requires at least Maven 3.3.1 -->
+                  <version>[3.3.1,)</version>
```

Comment:
Possibly some plugins used during build already require a newer version, but version 3.3.1 (released in 2015) added support for `.mvn/jvm.config`, so it is definitely the lower bound.